documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -36,6 +36,14 @@ export default {
 
 Path to your custom `wrangler.toml` or `wrangler.json` config file.
 
+### handlers
+
+Path to a file with additional [handlers](https://developers.cloudflare.com/workers/runtime-apis/handlers/) and (optionally) [Durable Objects](https://developers.cloudflare.com/durable-objects/) to be exported from the file the adapter generates. This allows you to, for example, include handlers for scheduled or queue triggers alongside the fetch handler your SvelteKit app.
+
+> [!NOTE] The adapter expects the `handlers` file to have a default export. If you only want to export a Durable Object, add `export default {};` to the file.
```

Comment:
Should we split the durable objects into a separate file to avoid this pitfall?

Reply:
We may have to add a check ourselves since the bundling step will be removed and delegated to Cloudflare's wrangler (meaning the error would only surface when deploying).

Is it possible to do something like:
```js
if (handler) {
  try {
  const module = await import(handler);

  if (!('default' in module)) {
    throw new Error('You need a default export in the handlers file. Add `export default {};` to it.');
  }
}
```